nncf/common/pruning/utils.py

```diff
@@ -590,8 +590,13 @@ def get_input_masks(node: NNCFNode, graph: NNCFGraph) -> List[Optional[NNCFTenso
     :param graph: Graph to work with.
     :return: Input masks.
     """
+    retval = []
     input_masks = [input_node.data['output_mask'] for input_node in graph.get_previous_nodes(node)]
-    return input_masks
+    for input_mask in input_masks:
+        retval.append(input_mask[node.node_name] if isinstance(input_mask, dict) else input_mask)
+    if not retval:
+        retval = []
```

Comment:
```suggestion
```

Reply:
This part is needed to cover the empty case. We already discussed this in https://github.com/openvinotoolkit/nncf/pull/1244#discussion_r955842604, https://github.com/openvinotoolkit/nncf/pull/1244#discussion_r952379487

Reply:
not retval is always `[]` but ok, we will refactor it in separate PR